From: Mitsutoshi NAKANO <ItSANgo@gmail.com>
Date: Sat, 20 Jun 2015 22:42:38 +0900
Subject: fix printf(string) -> printf("%s", string)

* Wnn/uum/select_ele.c (display_all):
* Wnn/uum/screen.c (print_buf_msg):
* Wnn/uum/jutil.c (yes_or_no_or_newline):
fixed printf(string) -> printf("%s", string).


modified:   ChangeLog
modified:   ChangeLog.en
---
 ChangeLog            | 6 ++++++
 ChangeLog.en         | 6 ++++++
 Wnn/uum/jutil.c      | 2 +-
 Wnn/uum/screen.c     | 4 ++--
 Wnn/uum/select_ele.c | 4 ++--
 5 files changed, 17 insertions(+), 5 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index b633087..eacd194 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -3,6 +3,12 @@
   Japanese...
  */
 
+Sat, 20 Jun 2015 22:40:38  Mitsutoshi NAKANO  <ItSANgo@gmail.com>
+
+	* Wnn/uum/select_ele.c (display_all): fixed printf(msg) -> printf("%s", msg).
+	* Wnn/uum/screen.c (print_buf_msg): fixed printf(string) -> printf("%s", string).
+	* Wnn/uum/jutil.c (yes_or_no_or_newline): fixed printf(string) -> printf("%s", string).
+
 Sat, 20 Jun 2015 17:56:56  Mitsutoshi NAKANO  <ItSANgo@gmail.com>
 
 	* ChangeLog.en: added these messages.
diff --git a/ChangeLog.en b/ChangeLog.en
index db934ae..2f57a38 100644
--- a/ChangeLog.en
+++ b/ChangeLog.en
@@ -1,3 +1,9 @@
+Sat, 20 Jun 2015 22:40:38  Mitsutoshi NAKANO  <ItSANgo@gmail.com>
+
+	* Wnn/uum/select_ele.c (display_all): fixed printf(msg) -> printf("%s", msg).
+	* Wnn/uum/screen.c (print_buf_msg): fixed printf(string) -> printf("%s", string).
+	* Wnn/uum/jutil.c (yes_or_no_or_newline): fixed printf(string) -> printf("%s", string).
+
 Sat, 20 Jun 2015 17:56:56  Mitsutoshi NAKANO  <ItSANgo@gmail.com>
 
 	* ChangeLog.en: added these messages.
diff --git a/Wnn/uum/jutil.c b/Wnn/uum/jutil.c
index 6d81b79..7142662 100644
--- a/Wnn/uum/jutil.c
+++ b/Wnn/uum/jutil.c
@@ -571,7 +571,7 @@ yes_or_no_or_newline (string)
   int x;
   throw_c (0);
   clr_line ();
-  printf (string);
+  printf ("%s", string);
   flush ();
   for (;;)
     {
diff --git a/Wnn/uum/screen.c b/Wnn/uum/screen.c
index d3be2c1..ec519c4 100644
--- a/Wnn/uum/screen.c
+++ b/Wnn/uum/screen.c
@@ -10,7 +10,7 @@
  *                 1987, 1988, 1989, 1990, 1991, 1992
  * Copyright OMRON Corporation. 1987, 1988, 1989, 1990, 1991, 1992, 1999
  * Copyright ASTEC, Inc. 1987, 1988, 1989, 1990, 1991, 1992
- * Copyright FreeWnn Project 1999, 2000, 2002
+ * Copyright FreeWnn Project 1999, 2000, 2002, 2015
  *
  * Maintainer:  FreeWnn Project   <freewnn@tomo.gr.jp>
  *
@@ -547,7 +547,7 @@ print_buf_msg (msg)
 {
   push_cursor ();
   throw_c (0);
-  printf (msg);
+  printf ("%s", msg);
   pop_cursor ();
   flush ();
 }
diff --git a/Wnn/uum/select_ele.c b/Wnn/uum/select_ele.c
index 3d6def3..8e6f56e 100644
--- a/Wnn/uum/select_ele.c
+++ b/Wnn/uum/select_ele.c
@@ -10,7 +10,7 @@
  *                 1987, 1988, 1989, 1990, 1991, 1992
  * Copyright OMRON Corporation. 1987, 1988, 1989, 1990, 1991, 1992, 1999
  * Copyright ASTEC, Inc. 1987, 1988, 1989, 1990, 1991, 1992
- * Copyright FreeWnn Project 1999, 2000, 2002, 2012
+ * Copyright FreeWnn Project 1999, 2000, 2002, 2012, 2015
  *
  * Maintainer:  FreeWnn Project
  *
@@ -224,7 +224,7 @@ display_all (mm, cc)
 
   throw_c (0);
   clr_line ();
-  printf (msg);
+  printf ("%s", msg);
   for (k = dd[cc]; k < dd[cc + 1]; k++)
     {
       cp = findcp (k, cc);
